src/foam/nanos/auth/Group.js

```diff
@@ -185,18 +185,25 @@ if ( session != null ) {
       //populate AppConfig url with request's RootUrl
       HttpServletRequest req = x.get(HttpServletRequest.class);
 
-      if ( req != null && ! SafetyUtil.isEmpty(req.getRequestURI()) ) {
+      if ( (req != null) && !SafetyUtil.isEmpty(req.getRequestURI()) ) {
```

Comment:
Why do we test for session above?  What happens when cron runs and there is no session?

Reply:
Great, one less thing.